src/scripts/settings/appSettings.js

```diff
@@ -20,6 +20,14 @@ define(['appStorage', 'events'], function (appStorage, events) {
         return this.get('enableAutoLogin') !== 'false';
     };
 
+    AppSettings.prototype.enableSystemExternalPlayers = function (val) {
+        if (val != null) {
```

Comment:
```suggestion
        if (val !== null) {
```

A strict comparison is usually better.

Reply:
good catch, missed that